addons/portal/views/inherited_mail_channel_view.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="mail.mail_channel_action_client_chat" model="ir.actions.client">
+        <field name="context">{'share_icon': True}</field>
```

Comment:
What is the purpose of that ?

Reply:
@tde-banana-odoo , When being in the discuss app,  we need a share icon and it's client action so need to pass share_icon in context to display an icon on systray.

>> clicking on the share button should open a wizard to invite people joining odoo :
https://drive.google.com/a/odoo.com/file/d/1YtV9c-9dOJHwjByu_wUE-nY5fg9Xv_Dk/view?usp=drivesdk